operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -380,6 +400,27 @@ public NextAction apply(Packet packet) {
         ScanCache.INSTANCE.registerScan(
             info.getNamespace(), info.getDomainUid(), new Scan(wlsDomainConfig, new DateTime()));
         packet.put(ProcessingConstants.DOMAIN_TOPOLOGY, wlsDomainConfig);
+        if (miiDomainZipHash != null) {
+          packet.put(ProcessingConstants.DOMAIN_HASH, miiDomainZipHash);
+        }
+        if (miiModelSecretsHash != null) {
+          packet.put(ProcessingConstants.SECRETS_HASH, miiModelSecretsHash);
+        }
+        String domainRestartVersion = info.getDomain().getRestartVersion();
+        String domainIntrospectVersion = info.getDomain().getIntrospectVersion();
+        int modelInImageSpecHash =  ConfigMapHelper.getModelInImageSpecHash(info.getDomain().getSpec().getImage());
+        if (domainRestartVersion != null) {
+          packet.put(ProcessingConstants.DOMAIN_RESTART_VERSION, domainRestartVersion);
+          data.put(ProcessingConstants.DOMAIN_RESTART_VERSION, domainRestartVersion);
+        }
+        if (domainIntrospectVersion != null) {
+          packet.put(ProcessingConstants.DOMAIN_INTROSPECT_VERSION, domainIntrospectVersion);
+          data.put(ProcessingConstants.DOMAIN_INTROSPECT_VERSION, domainIntrospectVersion);
+        }
+        if (info.getDomain().getDomainHomeSourceType().equals("FromModel")) {
```

Comment:
less chance for NPE if you reverse the equals check and start with the constant.